src/super_gradients/training/sg_trainer/sg_trainer.py

```diff
@@ -1866,7 +1950,11 @@ def evaluate(
 
             if not silent_mode:
                 # PRINT TITLES
-                pbar_start_msg = f"Validation epoch {epoch}" if evaluation_type == EvaluationType.VALIDATION else "Test"
+                pbar_start_msg = "Validating" if evaluation_type == EvaluationType.VALIDATION else "Testing"
```

Comment:
Why did you change the PBar message ?

Reply:
It's to add the possibility of include `dataset_name`, otherwise we would get `Testing epoch X` for every test dataset, and I thought it would be nicer to show the dataset name as well.
Eventually, it just means that `Validation epoch X` will be replaced with `Validation epoch X`, and for test datasets, it will be `Testing dataset="dataset-name": epoch X` (like in the description of the PR)